securedrop/bin/test

```diff
@@ -9,12 +9,14 @@ ln -s /dev/urandom /dev/random
 
 x11vnc -display :1 -autoport 5901 -shared &
 
-touch tests/log/firefox.log
-function finish {
-  cp tests/log/firefox.log /tmp/test-results/logs/
-  bash <(curl -s https://codecov.io/bash)
-}
-trap finish EXIT
+if test "${CIRCLE_BRANCH}" ; then
```

Comment:
Not a blocker, but I think the `[ -n "${CIRCLE_BRANCH}" ]` notation is more clear than `test` and would slightly prefer that as it makes skimming code easier.

Reply:
right ! done :-)